src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/SurveySvcClient.java

```diff
@@ -28,6 +100,126 @@
    * @return SurveyClassifierTypeDTO details of selector type requested.
    * @throws RestClientException something went wrong making http call.
    */
-  SurveyClassifierTypeDTO requestClassifierTypeSelector(UUID surveyId, UUID classifierType)
-      throws RestClientException;
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  public SurveyClassifierTypeDTO requestClassifierTypeSelector(
+      final UUID surveyId, final UUID classifierType) throws RestClientException {
+
+    UriComponents uriComponents =
+        restUtility.createUriComponents(
+            appConfig.getSurveySvc().getRequestClassifierTypesPath(),
+            null,
+            surveyId,
+            classifierType);
+
+    HttpEntity<?> httpEntity = restUtility.createHttpEntity(null);
+
+    log.with("survey_id", surveyId.toString())
+        .with("classifier_type", classifierType.toString())
+        .debug("Requesting survey classifier type");
+    ResponseEntity<String> responseEntity =
+        restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
+
+    SurveyClassifierTypeDTO result = null;
+    if (responseEntity != null && responseEntity.getStatusCode().is2xxSuccessful()) {
+      String responseBody = responseEntity.getBody();
+      try {
+        result = objectMapper.readValue(responseBody, SurveyClassifierTypeDTO.class);
+      } catch (IOException e) {
+        String msg = String.format("cause = %s - message = %s", e.getCause(), e.getMessage());
+        log.error(msg);
+      }
+    }
+    return result;
+  }
+
+  private SurveyDTO getSurveyDtoFromResponseEntity(ResponseEntity<String> responseEntity) {
+    SurveyDTO result = null;
+    if (responseEntity != null && responseEntity.getStatusCode().is2xxSuccessful()) {
+      String responseBody = responseEntity.getBody();
+      try {
+        result = objectMapper.readValue(responseBody, SurveyDTO.class);
+      } catch (IOException e) {
+        String msg = String.format("cause = %s - message = %s", e.getCause(), e.getMessage());
+        log.error(msg);
+      }
+    }
+    return result;
+  }
+
+  /**
+   * Get classifier type selector for Survey UUID and ClassifierType UUID.
+   *
+   * @param surveyId UUID for which to request classifiers.
+   * @return SurveyDTO representation of survey
+   * @throws RestClientException something went wrong making http call.
+   */
+  @Retryable(
```

Comment:
The `@Retryable` annotation should be used very sparingly. We can't have users waiting 25 seconds for a response.